resources/views/pages/campaign_inbox.blade.php

```diff
@@ -0,0 +1,13 @@
+@extends('layouts.master')
+
+@section('main_content')
+
+    @include('layouts.header', ['header' => 'Campaign Inbox', 'subtitle' => 'all pending reportbacks for the campaign'])
+
+	    <div class="container -padded">
+	        <div id="inboxContainer" class="wrapper">
+	            shove the react here
```

Comment:
Nitpicky thing, but this should probably say "loading..." or something more user friendly as a user could see this briefly on the page.